dataikuapi/dss/admin.py

```diff
@@ -946,3 +946,24 @@ def get_raw(self):
         Gets the whole status as a raw dictionary.
         """
         return self.status
+
+
+class DSSInstanceVariables(dict):
+    """
+    Dict containing the instance variables. The variables can be modified directly in the dict and persisted using its `save` method.
+
+    Do not create this directly, use :meth:`dataikuapi.dssclient.DSSClient.get_global_variables`
+    """
+
+    def __init__(self, client, variables):
+        super().__init__()
```

Comment:
This syntax doesn't seem to work in python 2.7. I think we still want to support python 2.7, so you'll probably need to stick with `super(dict, self).__init__()` (I think that still works in python 3)